<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Excel_Import</class>
 <widget class="QWidget" name="Frm_Excel_Import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel 导入</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DB Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel文件路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_directory">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_db_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_import">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Close">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Close</sender>
   <signal>clicked()</signal>
   <receiver>Frm_Excel_Import</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_select</sender>
   <signal>clicked()</signal>
   <receiver>Frm_Excel_Import</receiver>
   <slot>sel_directory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_import</sender>
   <signal>clicked()</signal>
   <receiver>Frm_Excel_Import</receiver>
   <slot>baidu_dl_pic01()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sel_directory()</slot>
  <slot>baidu_dl_pic01()</slot>
 </slots>
</ui>
